src/main/java/com/yahoo/sketches/frequencies/ItemsSketch.java

```diff
@@ -517,6 +526,47 @@ public int compareTo(final Row<T> that) {
     }
   }
 
+  @Override
```

Comment:
This is wrong, but the reason is quite subtle.  Equals() and hashCode() are implemented when it is desirable to create hashTables of these objects or attempt to sort them.   But sketches are stochastic not deterministic processes.  This means that even if you feed the same identical stream of inputs into two different sketches of the same class, configured the same way, there is no guarantee that the internal contents of the two sketches will be the same (there are some exceptions, but it is better not to expect equality).  This is because random noise is intentionally inserted into sketch algorithms and is key to how they work.  Thus, it is not possible to construct an algorithm that can guarantee equality.  They may be similar, but not equal (but what good is that?).  Thus overriding equals() and hashCode() makes no sense.  That means creating HashTables or attempting to sort sketches also makes no sense.  The Identity case, where A.equals(A) will work and the Object's equals() and hashCode() already handle that just fine, but it is not very useful.  :)